content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,172 @@
+
+### 💾 File system integration and file handling
+
+In Excalidraw, we use the [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) library to integrate with the file system of the operating system. This allows us to support a true open→edit→save workflow with proper over-saving and save-as on supported browsers, with a fallback to file uploads and downloads on other browsers. Read more about this feature in [tomayac](https://github.com/tomayac)'s [earlier article](/browser-nativefs/) on this blog. We also have [experimental support](https://web.dev/file-handling/#demo) for file type association, so that when you double-click an `.excalidraw` file in your file explorer, the Excalidraw PWA opens.
+
+### ⚙️ Gatsby plugin
+
+We also have a plugin for Gatsby that automatically converts links to saved Excalidraw drawings to inline SVG at build time. We are actually using it in this post for our charts. Implemented by [trevorblades](https://github.com/trevorblades) & [j-f1](https://github.com/j-f1), and you can find it under [@excalidraw/gatsby-embedder-excalidraw](https://github.com/excalidraw/gatsby-embedder-excalidraw).
+
+https://twitter.com/Vjeux/status/1257906664239333376
+
+### 📦 npm package
+
+One of the last things we've introduced this year was a completely new npm package, available at [`@excalidraw/excalidraw`](https://www.npmjs.com/package/@excalidraw/excalidraw). A long time in the making (thanks to [@aakansha1216](https://twitter.com/aakansha1216) for most of the work), this package allows you to easily embed Excalidraw as a React component into your apps.
+
+### 🌒 Dark Mode
+
+https://twitter.com/Msieur_Jo/status/1245288337897914373
+
+### 🏹 Arrow Binding
+
+https://twitter.com/excalidraw/status/1292403762427039744
+
+## Excalidraw in the news
+
+- https://pakstech.com/blog/draw-diagrams/
+- https://dev.to/ndsn/why-excalidraw-is-mightier-than-the-pen-and-the-sword-329f
+- https://news.ycombinator.com/item?id=23525648
+- https://news.ycombinator.com/item?id=22663435
+- https://web.dev/browser-nativefs/
+- https://www.producthunt.com/posts/excalidraw
+- React Europe talk https://www.youtube.com/watch?v=fix2-SynPGE
+
+- Another book: https://twitter.com/dchest/status/1264237749642637312
+- Used to illustrate an O'Reilly Book: https://www.amazon.com/Building-Serverless-Applications-Google-Cloud/dp/1492057096 https://twitter.com/wietsevenema/status/1253752608671621124
+
+## Get involved
+
+Excalidraw wouldn't have become what it is today without all the wonderful contributions. If you haven't already, you can start today! And remember, it's not just code that makes Excalidraw better. Every bit helps, be it bug reports, translations, suggestions for improvements, or just hanging out on our [Discord chat](https://discord.com/invite/UexuTaE).
```

Comment:
should we a link to [good first issues](https://github.com/excalidraw/excalidraw/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) ?

Reply:
probably we should mention here to join follow the twitter account as the header says `Get involved`

Reply:
```suggestion
Excalidraw wouldn't have become what it is today without all the wonderful contributions. If you haven't already, you can start today! And remember, it's not just code that makes Excalidraw better. Every bit helps, be it bug reports, translations, suggestions for improvements, or just hanging out on our [Discord chat](https://discord.com/invite/UexuTaE). Also don't forget to follow us on Twitter [@excalidraw](https://twitter.com/excalidraw) for all the latest news and announcements.
```